newsletter/__init__.py

```diff
@@ -0,0 +1,5 @@
+from django.utils.version import get_version
```

Comment:
This breaks the installer if Django is not yet installed (so for a fresh install of all your requirements for instance).

Since `setup.py` is used to install, and it imports newsletter, it will crash on the `django` import.

I have fixed this for `django-choices` the following way: https://github.com/bigjason/django-choices/commit/b95612edf545fef35dbcc47958ee578a61aa2377